solar_system.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'planet'
+
+class SolarSystem
+  attr_reader :star_name, :planets
+
+
+  def initialize(star_name)
+    @star_name = star_name
+    @planets = []
+  end
+
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+
+  def list_planets
+    list = ""
+    @planets.length.times do |i|
+      list += "\n#{i + 1}. #{@planets[i].name} "
+    end
+
+    return "Planets orbiting #{@star_name} #{list}"
+  end
+
+  def find_planet_by_name(planet_name)
+    x = ""
+
+    @planets.length.times do |i|
+      if planet_name.downcase == @planets[i].name.downcase
+        # matching_planet = @planets[i]
+        #cant figure out how to call summary method
```

Comment:
This definitely solves the problem, but we could clean this code up a little by using an enumerable:

```ruby
def find_planet_by_name(planet_name)
  planet_name = planet_name.downcase
  return @planets.find { |planet| planet.name.downcase == planet_name }
end
```